proto/semantic.proto

```diff
@@ -174,3 +178,30 @@ message Span {
   Position start = 1;
   Position end = 2;
 }
+
+message ScopeGraphFile {
+  string path = 1;
+  string language = 2;
+  repeated ScopeGraphNode nodes = 3;
+  repeated ScopeGraphPath paths = 4;
+  repeated ParseError errors = 5;
+}
+
+message ScopeGraphNode {
+  int32 id = 1;
```

Comment:
FYI Haskell native `Int` is 64-bits wide, so we could have slightly more efficient conversion (no `fromIntegral` upcasting) if we went with `int64`.